developer-docs-site/docs/guides/basics-life-of-txn.md

```diff
@@ -216,7 +219,7 @@ Mempool is a shared buffer that holds the transactions that are “waiting” to
 
 ### 1. REST Service → Mempool
 
-* After receiving a transaction from the client, the REST service proxies the transaction to a validator fullnode. The transaction is then sent to the validator node’s mempool.
+* After receiving a transaction from the client, the REST service sends the transaction to its own mempool, which then shares the transaction with the mempool of a validator fullnode. The mempool on the validator fullnode then shares the transaction with the mempool of a validator.
```

Comment:
This is likely out of the scope of this PR. Still, I want to learn and help:
Is the transaction processed in any way by the validator fullnode's mempool. If so, how? If not, why not send the transaction straight to the validator? What is the purpose of this proxy? Load balancing? Some checking?

We don't seem to describe the purpose of a validator fullnode here:
https://aptos.dev/nodes/validator-node/operator/node-requirements

And we don't even mention them here:
https://aptos.dev/concepts/validator-nodes/

 I am happy to help with updates to those pages once I have a greater understanding of VFNs.